Symfony/CS/Tests/Fixer/AbstractFixerTestBase.php

```diff
@@ -37,6 +37,20 @@ protected function getTestFile($filename = __FILE__)
         return $files[$filename];
     }
 
+    /**
+     * Tests if a fixer fixes a given input string to match the expected result. It is used both if you want to test
```

Comment:
I think the first line should a short description which may not expend to the next line (I think it is a PSR 5 rule).